.travis.yml

```diff
@@ -1,22 +1,23 @@
 language: minimal
 if: head_branch =~ ^v1.1beta_webportal_oci_spec_update$ OR branch = v1.1beta_webportal_oci_spec
 
+
 services:
   - docker
   
 before_install:
 
 script:
-   - cd ./spec/agent && docker build -t enterpriseconnect/agent:v1.1beta --build-arg EC_REV_V1_1BETA=${EC_REV_V1_1BETA} .
+   - cd ./spec/web-ui && docker build -t enterpriseconnect/webui:v1.1beta --build-arg WEBUI_BRANCH=${WEBUI_BRANCH} .
```

Comment:
suggest to do a quick ```docker run``` to prove the local tag is working prior to the docker push/deploy.

Reply:
Added docker run command after build to check if image is working